app/domain/operations/families/verifications/dmf_determination/build_cv3_family_payload_for_dmf.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require 'dry/monads'
+require 'dry/monads/do'
+
+module Operations
+  module Families
+    module Verifications
+      module DmfDetermination
+        # operation to build cv3 family to publish to fdsh
+        class BuildCv3FamilyPayloadForDmf
+          include Dry::Monads[:result, :do]
+          include ::Operations::Transmittable::TransmittableUtils
+
+          # @return [ Cv3Family ] Job successfully completed
+          def call(family, transmittable_params)
+            @job = transmittable_params[:job]
+            @transmission = transmittable_params[:transmission]
+            @transaction = transmittable_params[:transaction]
+            @family = family
+
+            cv3_family = yield build_cv3_family
+            valid_cv3_family = yield validate_cv3_family(cv3_family)
+
+            yield validate_all_family_members(valid_cv3_family)
+            result = yield confirm_transmittable_payload(cv3_family)
+
+            Success(result)
+          end
+
+          private
+
+          def build_cv3_family
+            cv3_family = Operations::Transformers::FamilyTo::Cv3Family.new.call(@family)
+            return handle_dmf_failure("Unable to transform family into cv3_family: #{cv3_family.failure}", :build_cv3_family) if cv3_family.failure?
+
+            cv3_family
+          end
+
+          def validate_cv3_family(cv3_family)
+            valid_cv3_family = AcaEntities::Operations::CreateFamily.new.call(cv3_family)
+            return handle_dmf_failure("Invalid cv3 family: #{valid_cv3_family.failure}", :validate_cv3_family) if valid_cv3_family.failure?
+
+            valid_cv3_family
+          end
+
+          def validate_all_family_members(aca_family)
+            invalid_persons = aca_family.family_members.map do |aca_member|
+              family_member = @family.family_members.detect { |fm| fm.hbx_id == aca_member.hbx_id }
+
+              # check if member does not have an enrollment
+              unless member_eligible?(aca_member)
```

Comment:
can you move this part of the code to new operation in `Operations::Fdsh::PayloadEligibility::`.

Reply:
Going to revisit on when handling dmf response